src/App.tsx

```diff
@@ -1,173 +1,63 @@
-import { Loader } from './components/Loader';
+import { FC } from 'react';
+import {
+  NavLink,
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import cn from 'classnames';
```

Comment:
It is better not to shorten the names to unreadable ones (`cn` -> `classNames`).
This applies to all code.